platform/darwin/src/MGLSource.h

```diff
@@ -1,8 +1,15 @@
 #import <Foundation/Foundation.h>
-#import <CoreGraphics/CoreGraphics.h>
 
+/**
+ A source supplies data to be shown on the map. Sources don't contain styling details like color or width. Use style layers to give visual representation to sources. 
```

Comment:
Refer to "subclasses of `MGLStyleLayer`" instead of "style layer" so developers can easily navigate in the documentation.